src/MSALWrapper/PCAWrapper.cs

```diff
@@ -86,6 +86,13 @@ public interface IPCAWrapper
         /// </returns>
         Task<TokenResult> GetTokenDeviceCodeAsync(IEnumerable<string> scopes, Func<DeviceCodeResult, Task> callback, CancellationToken cancellationToken);
 
+        /// <summary>

```

Comment:
I think these changes to this file should happen in their own Pull request.

Reply:
Can you approve Rick's PR then? He already has these changes in his PR. I can merge main after that and it will be removed from my PR? This file is a blocker to my PR.

Reply:
Spoke offline and Rick's PR is now merged anyway.